proxy/README.md

```diff
@@ -1,33 +1,26 @@
-# AppProxy
+# Proxy
 
-Intermediate serving a specialised API to the radicle-upstream frontend via
-GraphQL.
+A background service that implements all business logic tying together the
+radicle protocol and radicle registry. It provides a GraphQL API to the
+frontend.
 
-### Run
-
-To start up the binary you can run: `cargo run -- local`. After that the API is
-served on `http://127.0.0.1:8080/graphql`.
 
-To use an in-memory Radicle client, run `cargo run -- memory`.
-
-You can open the GraphQL explorer in the browser at `http://localhost:8080`.
+### Run
 
-To use the `radicle-registry` node as the backend for the proxy run `cargo run
--- registry`. See [`radicle-registry`][run-registry] for information on how to
-build and run the registry node.
+To start up the binary you can run: `cargo run -- --source=memory`.
+After that the API is served on `http://127.0.0.1:8080/graphql`.
 
-[run-registry]: https://github.com/radicle-dev/radicle-registry#building-and-running-the-node
 
 ### Testing
 
 Before running the test suite, download the test fixtures:
 
-```bash
-$ git submodule update --recursive
+```sh
+git submodule update --recursive
```

Comment:
This part seems to be shared between all tests in the repo, so might be better placed in the root readme.

Reply:
I think it would make the docs more confusing. If we keep the Rust and JS development sections in their respective READMEs and extract the submodule update step to the top-level, then someone reading this testing section might miss that part and wonder why the tests are not working.